pkg/proto/nbcontract/v1/utils_test.go

```diff
@@ -176,3 +176,98 @@ func Test_getKubeletNodeLabels(t *testing.T) {
 		})
 	}
 }
+
+func TestGetOutBoundCmd(t *testing.T) {
```

Comment:
Which one were you comparing with? This? https://github.com/Azure/AgentBaker/blob/e1d423e9731b222f30a1b469c170db58b88c5530/pkg/agent/variables.go#L177
It seems to me they are the same?

Reply:
Oh I think it used to be different in an older version: https://github.com/Devinwong/karpenter-provider-azure/blob/d731ed0dfc4855d7fac165ce506c3fa1678d87f1/pkg/providers/imagefamily/bootstrap/helpers.go#L69

I think it was because you were trying to see if this could be computed on the VHD instead, which I also looked into, but we would probably need to add more variables to the contract. So I just updated it to be the same as the current one in the AB and decided it probably needs to be passed in from the bootstrapper.

Reply:
I see.